src/rpp/mutex.cpp

```diff
@@ -3,6 +3,14 @@
     #define WIN32_LEAN_AND_MEAN
     #include <Windows.h>
 #endif
+#if RPP_FREERTOS
+    #include "FreeRTOS.h"
```

Comment:
This seems like incorrect include style, since there are no "FreeRTOS.h" etc right next to these files.
If the includes are defined by platform includes, it should be using <FreeRTOS.h>, to show that external search path is preferred.
Modern compilers fall back to global search, but semantically <> means external, while "" should mean relative.